FLEDGE.md

```diff
@@ -803,3 +811,184 @@ These signals were requested in [issue 435](https://github.com/WICG/turtledove/i
+  'owner': 'https://www.example-dsp.com',
+  'name': 'womens-running-shoes',
+  'lifetimeMs': 30 * kSecsPerDay,
+  'updateURL': 'https://www.example-dsp.com/update?id=12345', //optional
+  'additionalBidKey': 'EA/fR/uU8VNqT3w/2ic4P6Azdaj1J8U35vFwPEf5T4Y='
+};
+navigator.joinAdInterestGroup(myGroup);
+```
+
+##### 6.2.2 How Additional Bids Specify their Negative Interest Groups
+
+Additional bids specify the negative interest groups they're negatively targeting against using one of two fields in their JSON data structure - negativeInterestGroup or negativeInterestGroups.
+
+If an additional bid only needs to specify a single negative interest group, it can do so as follows:
+
+```
+const additionalBid = {
+  // ...
+  "negativeInterestGroup": "example_advertiser_negative_interest_group"
+  // ...
+}
+```
+
+If an additional bid needs to specify two or more negative interest groups, all of those negative interest groups must be joined from the same site, and that site must be identified ahead of time in the additional bid using the joiningOrigin field:
+
+```
+const additionalBid = {
+  // ...
+  "negativeInterestGroups": {
+    joiningOrigin: "https://example-advertiser.com",
+    interestGroupNames: [
+      "example_advertiser_negative_interest_group_a",
+      "example_advertiser_negative_interest_group_b",
+    ]
+  },
+  // ...
+}
+```
+
+Any negative interest group that wasn't joined from that identified site won't be considered for negative targeting. This restriction is enforced so that negative targeting can only use targeting data from a single site.  An additional bid that only specifies one negative interest group is not subject to the same restriction on joining origin.
+
+##### 6.2.3 Additional Bid Keys
+
+We use a cryptographic signature mechanism to ensure that only the owner of a negative interest group can use it with additional bids. Each buyer will need to create a Ed25519 public/secret key pair to sign their additional bids to prove their authenticity, and to regularly rotate their key pairs.
+
+When a buyer joins a user into a negative interest group, they must provide their 32-byte Ed25519 public key, expressed as a base64-encoded string, via the negative interest group's additionalBidKey field. This can be seen in the example above in section 6.2.1 Negative Interest Groups. The additionalBidKey can be then updated via the negative interest group's updateURL, for example, to enable a buyer to rotate their Ed25519 key pair faster than they could with the expiration of their negative interest groups alone.
+
+When the buyer issues an additional bid, that bid needs to be signed using their Ed25519 secret key. During a key rotation, the buyer may need to provide a signature of the additional bid with both the old and the new additionalBidKeys while negative interest groups stored on users' devices are updated to the new key. It's for this reason that additional bids may have more than one signature provided alongside the bid.
```

Comment:
>the buyer may need to provide a signature of the additional bid with both the old and the new additionalBidKeys while negative interest groups stored on users' devices are updated to the new key

Does the provision of dual keys 'trigger' a call to the optional `updateURL`, or does this happen in the normal PAAPI course of it being called once daily sometime after a bid win?

Reply:
Negative interest groups are updated at the same time and in the same way as regular interest groups. I've added some text here to clarify.